experimental/pg/sql/rikai--0.1.sql

```diff
@@ -0,0 +1,61 @@
+-- Create Schema to contain all Rikai functionality
+CREATE SCHEMA IF NOT EXISTS ml;
+
+-- Semantic Types
+CREATE TYPE image AS (uri TEXT);
+
+CREATE TYPE detection AS (label TEXT, label_id int, box box, score real);
```

Comment:
where's the box type?

Reply:
Box and Polygon `Mask in rikai` are already supported. https://www.postgresql.org/docs/current/datatype-geometric.html